### ISSUE TITLE
Add avatars to frontpage preview of blog posts and Twitter posts

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/nextcloud.com#155]
Having faces there is very important to not just have bland text. cc @jospoortvliet @Espina2 @LukasReschke

**Comment by Espina2** [comment]
Temporary solution for now, not everything is finishes on the page.

**Comment by jancborchardt** [comment]
Of course, just opening issues where I see things. Nothing is ever done ;)
